src/ValuedTextInput/valuedTextInput.css

```diff
@@ -16,10 +18,15 @@
         }
     }
 
+    &::placeholder
+    {
+        font:           inherit;
```

Comment:
here `&` is not the input itself (it’s the outer container div), so `placeholder` doesn’t make sense